README.md

```diff
@@ -1,7 +1,7 @@
 # SAS-Data
-This is the repository found at [Wireless Innovation Forum](https://github.com/Wireless-Innovation-Forum/SAS-Data). 
-It holds all data related to the [Wireless Innovation Forum / Spectrum Access System](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System) 
-GitHub repository, which are stored in Git LFS (Large File storage).
+This is the repository found at [Wireless Innovation Forum](https://github.com/Wireless-Innovation-Forum/Common-Data). 
+It holds all data related to the [Wireless Innovation Forum / Spectrum Access System and Automated Frequency Coordination systems](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System, https://github.com/Wireless-Innovation-Forum/6-GHz-AFC) 
```

Comment:
Because these [name](links) are markdown links, please use instead the following so there are properly rendered.
```
It holds all data related to the [Spectrum Access Systems](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System) 
and  [Automated Frequency Coordination systems](https://github.com/Wireless-Innovation-Forum/6-GHz-AFC) 
```